localization/english/dvg_flavor_oceania_l_english.yml

```diff
@@ -0,0 +1,18 @@
+﻿l_english:
+ #State Traits
+ state_trait_dvg_australia_locked: "Australia Gamerule"
```

Comment:
Maybe just Australian Colonization or something?
In Game rules they can define the gamerule.